molten-http-client/src/test/java/com/hotels/molten/http/client/ReactorNettyHttpClientTest.java

```diff
@@ -75,6 +76,7 @@ public void shouldWork() throws InterruptedException {
             .metrics(true)
             .build();
         HttpClient.create(connectionProvider)
+            .protocol(HttpProtocol.H2C)
```

Comment:
I don't think we need to specify the protocol in this test (including the other test case, and the `OkHttpTest`), since it's just a general client test to see if the underlying client is working without molten.

Reply:
Removed